exec/wd.c

```diff
@@ -585,7 +585,11 @@ static void wd_scan_resources (void)
 static void watchdog_timeout_apply (uint32_t new)
```

Comment:
Please document this function, such as it's parameters, and a description of what actions it takes and why it might be called.